spec/components/provider_interface/course_details_component_spec.rb

```diff
@@ -120,7 +150,7 @@ def row_text_selector(row_name, render)
     render_text = row_text_selector(:location, render)
 
     expect(render_text).to include('Location')
-    expect(render_text).to include('First Road (F34)')
```

Comment:
same as above let's not change this as this is live code and not behind a feature flag